### PR TITLE
Feat/add slint error relative paths

### PR DESCRIPTION
**Pull request by mariaozamiz**
### :pushpin: References

-   **Issue:** Closes [Add Eslint error for relative paths](https://app.clickup.com/t/8694bruua)

### :memo: Implementation

Add error alert from eslint to implement absolute paths in project. 

### :video_camera: Screenshots/Screen capture
Before lint -fix
Captura de pantalla 2024-04-19 a las 15 26 22

After fix
Captura de pantalla 2024-04-19 a las 15 26 40

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -57,9 +57,13 @@
...
         "testing-library/no-debugging-utils": "off",
-        "testing-library/no-dom-import": "off"
+        "testing-library/no-dom-import": "off",
+        "no-relative-import-paths/no-relative-import-paths": [
+            "error",
+            { "allowSameFolder": true, "rootDir": "src", "prefix": "$" }
+        ]
     },
-    "plugins": ["@typescript-eslint", "react-hooks", "unused-imports"],
+    "plugins": ["@typescript-eslint", "react-hooks", "no-relative-import-paths", "unused-imports"],
```
[the remark this answers is not in the document]
Comment (mariaozamiz) [reply .eslintrc.json:66]: I had mine set to "shortest" and was working fine with the shortcut, but I think is a good idea to add a vscode settings file so we all can align to "non-relative" for consistency

[the remark this answers is not in the document]
Comment (tokland) [reply .eslintrc.json:66]: > I had mine set to "shortest" and was working fine with the shortcut, but I think is a good idea to add a vscode settings file so we all can align to "non-relative" for consistency

"shortest" is the default, but, for some reason, only "non-relative" does an import using $ for me:

```
import { ValidationRuleGroup } from "../entities/ValidationRuleGroup"; // shortest
import { ValidationRuleGroup } from "../entities/ValidationRuleGroup"; // relative
import { ValidationRuleGroup } from "../entities/ValidationRuleGroup"; // project-relative
import { ValidationRuleGroup } from "$/domain/entities/ValidationRuleGroup"; // non-relative
```

Does it work differently for you? In any case, my point was that, to add `.vscode` to the repo.

[the remark this answers is not in the document]
Comment (mariaozamiz) [reply .eslintrc.json:66]: Right, for me both "non-relative" and "shortest" show identical results when using the import shortcut. The settings for vscode has been added in this commit: [chore: add vscode settings with non-relative set for imports](https://github.com/EyeSeeTea/data-quality-dev/pull/52/commits/5be52157790b52e6e57a9335017c233cbab6ea56)

**Review by tokland — APPROVED** [review]
I checked only the absolute-path imports + lint + vscode config, all good by me!

**Pull request merged**